osquery/tables/applications/windows/carbon_black.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/tables.h>
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/tables/system/windows/registry.h>
+
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/algorithm/string/replace.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+
+namespace fs = boost::filesystem;
+
+namespace osquery {
+namespace tables {
+
+// Carbon Black registry path 
+#define kCbRegLoc "SOFTWARE\\CarbonBlack\\config"
+// Path to Carbon Black direcotry
+#define kCbDir "C:\\Windows\\CarbonBlack\\"
```

Comment:
Would it be better to use `%SystemRoot%\CarbonBlack\` or `%windir%\CarbonBlack\` instead?